apollo-adminservice/src/main/java/com/ctrip/framework/apollo/adminservice/AdminServiceAutoConfiguration.java

```diff
@@ -45,4 +49,19 @@ public FilterRegistrationBean<AdminServiceAuthenticationFilter> adminServiceAuth
 
     return filterRegistrationBean;
   }
+
+  /**
+   * for apollo-assembly
+   */
+  @Order(99)
+  @Profile("assembly")
+  @Configuration
+  static class AdminServiceSecurityConfigurer extends WebSecurityConfigurerAdapter {
```

Comment:
Is the AdminServiceSecurityConfigurer just for assembly stuff? If that's true, maybe it'd make more sense to set it up in a file, something like AssemblyAutoConfiguration?